runtime/src/bank.rs

```diff
@@ -4929,6 +4930,11 @@ impl Bank {
             &mut executed_units,
         );
         process_message_time.stop();
+        executed_us = timings
+            .execute_accessories
+            .process_instructions
+            .total_us
+            .saturating_sub(executed_us);
```

Comment:
I forget all these precise metrics definitions, but, why `process_message_time.as_us()` isn't used just like 4 lines below (L4941)?

Reply:
changes of `timings.execute_accessories.process_instructions.total_us` strictly capture  `invoke_context.process_instruction`, so does `accumulated_consumed_units`; it is more accurate than `process_message_time` when used to calculate cu/time ratio with `accumulated_consumed_units`